AppController/terminate.rb

```diff
@@ -54,7 +54,12 @@ def self.disable_database_writes
     `/root/appscale/AppDB/cassandra/cassandra/bin/nodetool -h #{ip} -p 7070 drain`
 
     # Next, stop ZooKeeper politely.
-    `service zookeeper-server stop`
+    result = `service --status-all 2> /dev/null|grep zookeeper-server`
+    if result != 1
+      `service zookeeper stop`
```

Comment:
Good idea: although in terminate no other commands return an error. So I added the check upon starts: it will raise an exception if we cannot start zookeeper.